docs/streams/kafka_on_gdn/kafka-nodejs-client.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_position: 50
+title: Kafka NodeJS Client
+---
+
+The Kafka-Node client can be used to create Kafka producers, consumers, and topics in Node.js. Full documentation [here](https://www.npmjs.com/package/kafka-node).
+
+Note: Recommend to use [kafkajs](../kafkajs/README.md) client instead, because `kafka-node` has not been supporting since 2019 year.
```

Comment:
```suggestion
:::note
We recommend using [kafkajs](../kafkajs/README.md) client instead, because `kafka-node` has not been supporting since 2019.
:::
```

Reply:
In the note, add , "For more information about using kafkaJS with Macrometa, refer to [KafkaJSclient](kafkajs-client.md)."